### ISSUE TITLE
Feature Request/Idea: Allow to choose which fields should be pre-filled in datasets or collections

### ISSUE THREAD

**Issue by tjouneau** [IQSS/dataverse#8653]
**Overview of the Feature Request**
Currently many fields (name, affiliation, dataverse name...) are pre-filled with information from the user's profile.
While this feature makes sense in most cases, it would be nice to be able to switch it on or off depending on the needs, for ex. through a parameter configurable from api/admin/settings.

**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
All

**What inspired the request?**
We would have liked to have the affiliation field blank at dataset creation. But I feel it's only a specific need into a larger and more generic possible functionality.

**What existing behavior do you want changed?**
See overview.

**Any brand new behavior do you want to add to Dataverse?**
No, just configuration parameters.

**Any related open or closed issues to this feature request?**

**Comment by pdurbin** [comment]
@tjouneau thanks for creating this issue and the discussion at https://groups.google.com/g/dataverse-community/c/iuqlqVYiEqA/m/d0-agmUqKgAJ

Just to be clear, would a system-wide setting be sufficient for your needs? Would the system-wide setting cover both dataset and collection creation? Or would these be separate settings? Would the setting allow you to pick which fields are prepopulated? Or would the setting simply turn off or on prepopulation of the existing fields?

**Comment by pdurbin** [comment]
@tjouneau just checking if you saw my questions above or have any other details to share. Thanks.

**Comment by tjouneau** [comment]
Hi Philip, sorry for having let this unattended.
I think that the suggestion above still makes sense.
As for your questions : 
- yes, it would be optimal to pick up individually which fields are pre-populated
- my initial need concerned datasets but I guess being able to do this for collections would only be consistent
FYI as my institutional repo will merge into another, larger one (Recherche Data Gouv), I'm not sure I will be able to test this once it is done. It would be nice of course to know if any other user sees the use of such a feature. But as I said, I think it makes sense and deserves a look, independently from my particular needs.

**Comment by CB-HAL** [comment]
We would support such a feature. We need it for the “Deposit Date”, we don’t use this field and must delete it every time when we add a dataset. This would not be the greatest problem, but sometimes people forget to delete it and we have to rework the metadata.
